src/AutoRest.CSharp/AutoRest/Plugins/CSharpProj.cs

```diff
@@ -29,6 +28,21 @@ internal class CSharpProj : IPlugin
     <PackageReference Include=""Azure.Core"" Version=""1.11.0"" />
   </ItemGroup>
 
+</Project>
+";
+        private string _armCsProjContent = @"<Project Sdk=""Microsoft.NET.Sdk"">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <Nullable>annotations</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include=""Azure.Core"" Version=""1.10.0"" />
+    <PackageReference Include=""Azure.ResourceManager.Core"" Version=""1.0.0-alpha.20210325.1"" />
```

Comment:
If you are just adding a reference to `Azure.ResourceManager.Core`, you should just add a single part like I do with `_lowLevelExperimentalReference`, that way we don't duplicate the entire csproj contents for one ref.